v1.0/configure-replication-zones.md

```diff
@@ -495,6 +517,7 @@ There's no need to make zone configuration changes; by default, the cluster is c
 
 1. Start each node with a different locality attribute:
 
+    {% include copy-clipboard.html %}
```

Comment:
Does the copy work when there are multiple commands? I'm not sure...

Reply:
Yes. It copies the lines prefixed with `#` as well, but that's fine because shells will ignore those lines and execute the rest.

Reply:
Right, the comments aren't a problem but rather the multiple commands. All but the last command will attempt to execute and will fail (in this case, due to placeholder text). For example, here's what I get when I copy and paste into a terminal:

```
~/src/github.com/cockroachdb/cockroach$ # Start the two nodes in datacenter 1:
~/src/github.com/cockroachdb/cockroach$ cockroach start --insecure --host=<node1 hostname> --locality=datacenter=us-1
-bash: node1: No such file or directory
~/src/github.com/cockroachdb/cockroach$ cockroach start --insecure --host=<node2 hostname> --locality=datacenter=us-1 --join=<node1 hostname>:27257
-bash: node2: No such file or directory
~/src/github.com/cockroachdb/cockroach$
~/src/github.com/cockroachdb/cockroach$ # Start the two nodes in datacenter 2:
~/src/github.com/cockroachdb/cockroach$ cockroach start --insecure --host=<node3 hostname> --locality=datacenter=us-2 --join=<node1 hostname>:27257
-bash: node3: No such file or directory
~/src/github.com/cockroachdb/cockroach$ cockroach start --insecure --host=<node4 hostname> --locality=datacenter=us-2 --join=<node1 hostname>:27257
-bash: node4: No such file or directory
~/src/github.com/cockroachdb/cockroach$
~/src/github.com/cockroachdb/cockroach$ # Start the two nodes in datacenter 3:
~/src/github.com/cockroachdb/cockroach$ cockroach start --insecure --host=<node5 hostname> --locality=datacenter=us-3 --join=<node1 hostname>:27257
-bash: node5: No such file or directory
~/src/github.com/cockroachdb/cockroach$ cockroach start --insecure --host=<node6 hostname> --locality=datacenter=us-3 --join=<node1 hostname>:27257
```

For that reason, we usually try to avoid using copy-to-clipboard on code blocks including multiple commands. One approach is to split the commands into separate code blocks, each with a copy-to-clipboard include. The other approach is to just not use copy-to-clipboard on these examples. What are you thoughts?

Reply:
I should really update https://github.com/cockroachdb/docs/blob/master/CONTRIBUTING.md#code-samples to be more explicit. Opened an issue: https://github.com/cockroachdb/docs/issues/2688.